x-pack/plugins/synthetics/server/routes/monitor_cruds/edit_monitor.ts

```diff
@@ -224,7 +224,13 @@ export const syncEditedMonitor = async ({
     return {
       failedPolicyUpdates,
       publicSyncErrors,
-      editedMonitor: editedMonitorSavedObject,
+      editedMonitor: {
```

Comment:
"Update" is much more common as a term in APIs than "Edit". For instance the acronym CRUD (Create, Read, Update, Delete), and as another example the [ES Document Update API](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-update.html)

Thoughts on changing to Update? I think it just seems more industry standard / professional

Reply:
this part is not relevant to this PR change and also not returned to user.